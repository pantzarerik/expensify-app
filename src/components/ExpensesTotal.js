import React from 'react'
import expenses from '../tests/fixtures/expenses'

class ExpensesTotal extends React.Component {
  render() {
    return (
      
    )
  }
}

export default ExpensesTotal
