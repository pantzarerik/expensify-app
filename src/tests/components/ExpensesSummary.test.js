import React from 'react'
import { shallow } from 'enzyme'
import { ExpensesSummary } from '../../components/ExpensesSummary'

test('should show the ExpensesSummary component correctly with one expense', () => {
  const wrapper = shallow(<ExpensesSummary expenseCount={1} expensesTotal={235} />)
  expect(wrapper).toMatchSnapshot()
})

test('should show the ExpensesSummary component correctly with multiple expenses', () => {
  const wrapper = shallow(<ExpensesSummary expenseCount={3} expensesTotal={121200} />)
  expect(wrapper).toMatchSnapshot()
})
