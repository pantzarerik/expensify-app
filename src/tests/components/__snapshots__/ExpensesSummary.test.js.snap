// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should show the ExpensesSummary component correctly with multiple expenses 1`] = `
<div>
  <h1>
    Viewing 
    3
     
    expenses
     totalling 
    $1,212.00
  </h1>
</div>
`;

exports[`should show the ExpensesSummary component correctly with one expense 1`] = `
<div>
  <h1>
    Viewing 
    1
     
    expense
     totalling 
    $2.35
  </h1>
</div>
`;
